docs/ecosystem/predictionguard.md

```diff
@@ -0,0 +1,56 @@
+# Prediction Guard
+
+This page covers how to use the Prediction Guard ecosystem within LangChain.
+It is broken into two parts: installation and setup, and then references to specific Cohere wrappers.
```

Comment:
I'm assuming the `Cohere` reference is unintentional? (here and in the docstring below)

Reply:
@vowelparrot Ah thanks for catching this. I've updated the file.